Data.py

```diff
@@ -29,10 +29,12 @@ def __init__(self):
                                        self.hdf5_runs_path)
 
         # Load data indexes for training and validation
+        self.train_all_steer_path = ARGS.data_path + '/train_all_steer'
```

Comment:
Doesn't make sense to make these class variables if they are not used anywhere else, I would  make them local variables.